src/App.tsx

```diff
@@ -34,22 +34,24 @@ function App() {
 
   return (
     <>
-      <h1>Welcome to TypeFaceOff!</h1>
+      <h1>Welcome to TypefaceOff!</h1>
```

Comment:
Why is the "f" not capitalised anymore? Will this be the same across the rest of our project?

Reply:
Oh sorry, I accidently deleted the <h1> tag one time and forgot the layout of the text I can change it back